src/components/settings/ExportMPCAccount/index.tsx

```diff
@@ -0,0 +1,59 @@
+import CopyButton from '@/components/common/CopyButton'
+import useMPC from '@/hooks/wallets/mpc/useMPC'
+import { Alert, Box, Button, TextField, Typography } from '@mui/material'
+import { COREKIT_STATUS } from '@web3auth/mpc-core-kit'
+import { useState } from 'react'
+
+const ExportMPCAccount = () => {
+  const mpcCoreKit = useMPC()
+
+  const [pk, setPk] = useState<string | undefined>()
+
+  const isLoggedIn = mpcCoreKit?.status === COREKIT_STATUS.LOGGED_IN
+
+  const exportPK = async () => {
+    const exportedPK = await mpcCoreKit?._UNSAFE_exportTssKey()
```

Comment:
This call seems to take a few seconds for me. I would suggest adding a loading state and disable the Export button to give the user some indication.